sigs.yml

```diff
@@ -660,16 +660,16 @@
     gcLiaison:
       - name: 'Juraci Paixão Kröhling'
         github: jpkrohling
-  - name: Project Tooling and Infrastructure
+  - name: Project Infrastructure
     meeting: Weekly on Wednesday at 08:00 AM PT
     notes:
       type: gDoc
       value: 1_dTP2XIaZoFjNiipkeMaGJN10NrhS20FiBTTJoQJeVM
     chat:
       - type: slack
-        name: '#otel-project-tooling'
+        name: '#otel-project-infra'
         id: C07BPU981PV
-    invites: calendar-project-tooling
+    invites: calendar-project-infra
```

Comment:
I'll rename this once the PR is approved

Reply:
resolving this conversation to merge the PR